src/react/components/_finances/TransactionsCard/UnappliedPaymentsInfo.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import formatCurrency from 'functions/formatCurrency';
+import './UnappliedPaymentsInfo.scss';
+
+const UnappliedPaymentsInfo = ({ unappliedBalance }) => {
+  const [showMore, setShowMore] = useState(false);
+
+  if (unappliedBalance > 0) {
+    const linkText = showMore ? 'Show less' : 'What is this?';
+
+    return (
+      <div className="UnappliedPaymentsInfo">
+        <div className="UnappliedPaymentsInfo__header">
+          <h2>Unapplied Payments and Aid: {formatCurrency(unappliedBalance)}</h2>
+          <a onClick={() => setShowMore(!showMore)}>{linkText}</a>
+        </div>
+        {showMore && (
+          <div className="UnappliedPaymentsInfo__more">
+            Unapplied payments and aid do not help lower your Total Unpaid
```

Comment:
@developish, I'd prefer to see this message in the CS Message Catalog, instead of hardcoded here.  This way its a lot easier to change through config, instead of having to update this code.